index.html

```diff
@@ -22,47 +23,51 @@ <h1>Daniel Eicher</h1>
   </header>
   <figure>
     <img src="https://source.unsplash.com/random/600x300" alt="">
-    <figcaption>This is an example image</figcaption>
+    <figcaption>Here will be be a picture of mysfelf</figcaption>
   </figure>
   <section id="interests">
-    <h2>Interests:</h2>
+    <h2>Interests</h2>
     <ul>
       <li>Family</li>
       <li>Technology</li>
       <li>Nature</li>
     </ul>
   </section>
   <section id="hobbies">
-    <h2>Hobbies:</h2>
+    <h2>Hobbies</h2>
     <ul>
       <li>Hiking</li>
       <li>Coding</li>
       <li>Reading books</li>
     </ul>
   </section>
   <section id="contact">
-    <h2>Contact:</h2>
+    <h2>Contact</h2>
     <ul>
       <li><span class="bold">Phone: </span>+34 642 104 722</li>
       <li><span class="bold">Mail: </span><a href="mailto:danieleicher@gmx.net">danieleicher@gmx.net</a></li>
-      <li><span class="bold">Connect with me on:<span><br></b>
+      <li><span class="bold">Connect with me on:<span><br>
             <!-- GitHub-logo with a link to my profil -->
-            <a href="https://github.com/codingoak" target="_blank"><img
+            <a class="logo" href="https://github.com/codingoak" target="_blank"><img
                 src="https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg" width="32px"
                 height="32px" alt="github-logo"></a>
             <!-- CodePen-logo with a link to my profil -->
-            <a href="https://codepen.io/codingoak" target="_blank"><img
+            <a class="logo" href="https://codepen.io/codingoak" target="_blank"><img
                 src="https://res.cloudinary.com/css-tricks/images/w_32,h_32,c_fill/f_auto,q_auto/v1637703542/Button-Black-Large/Button-Black-Large-png?_i=AA"
                 width="32px" height="32px" alt="codepen-logo"></a>
             <!-- LinkedIn-logo with a link to my profil -->
-            <a href="https://www.linkedin.com/in/daniel-eicher-963844203/" target="_blank"><img
+            <a class="logo" href="https://www.linkedin.com/in/daniel-eicher-963844203/" target="_blank"><img
                 src="https://upload.wikimedia.org/wikipedia/commons/c/ca/LinkedIn_logo_initials.png" width="32px"
                 height="32px" alt="linkedin-logo"></a>
+            <!-- facebook-logo with a link to my profil -->
+            <a class="logo" href="https://www.facebook.com/daniel.eicher.355/"><img
+                src="https://upload.wikimedia.org/wikipedia/commons/5/51/Facebook_f_logo_%282019%29.svg" width="32px"
+                height="32px" alt="linkedin-logo"></a>
       </li>
     </ul>
   </section>
   <section id="biography">
-    <h2>Biography:</h2>
+    <h2>Biography</h2>
     <div>
```

Comment:
Maybe I oversee this but I can not find the closing tag for this <div> element...